### PR TITLE
Write doc for project members

### PR DESCRIPTION
**Pull request by 9a24f0**
Includes member listing and invite member.
Resolve GH-162

### CODE REVIEW

File: docs/source/meth/uc-anal-design/project.rst [hunk 1]
```diff
@@ -39,3 +39,60 @@ is described in :numref:`p-edit`.
...
+   :caption: Analysis sequence diagram for adding member
+      when user is existed in project.
+   :name: add-member-exist-name
+
+If added user is an assistant
+
+.. uml:: uml/p-member-add-exist-name.puml
+   :caption: Analysis sequence diagram for adding member
+      when user is an assistant.
+   :name: add-member-exist-name
```
Comment (Huy-Ngo) [inline docs/source/meth/uc-anal-design/project.rst:92]: The reference is duplicated and wrong here.

File: docs/source/meth/uc-anal-design/project.rst [hunk 2]
```diff
@@ -39,3 +39,60 @@ is described in :numref:`p-edit`.
 .. uml:: uml/p-edit.puml
    :caption: Activity diagram illustrating updating project's basic information
    :name: p-edit
```
Comment (Huy-Ngo) [inline docs/source/meth/uc-anal-design/project.rst:41]: Please add scale here so it doesn't overwrite the page number
```suggestion
.. uml:: uml/p-edit.puml
   :scale: 80%
   :caption: Activity diagram illustrating updating project's basic information
   :name: p-edit
```

**Review by Huy-Ngo — CHANGES_REQUESTED** [review]

File: docs/source/meth/uc-anal-design/project.rst (hunk 2 above)
Comment (9a24f0) [reply docs/source/meth/uc-anal-design/project.rst]: Addressed in c793fcc8f880213f76acf103aeaebfcc161a6efe

File: docs/source/meth/uc-anal-design/project.rst (hunk 1 above)
Comment (9a24f0) [reply docs/source/meth/uc-anal-design/project.rst]: Addressed in c793fcc8f880213f76acf103aeaebfcc161a6efe

**Comment by Huy-Ngo** [comment]
Please add this in the `latex_elements` of the `conf.py` file.

```py
'figure_align': 'H',
```

**Review by Huy-Ngo — APPROVED** [review]
It's ok now.

**Pull request merged**